doc/schema/schema.md

```diff
@@ -40,10 +41,20 @@ Currently there are 2 beta implementations of this design:
  <li><a class="reference internal" href="#examples">Examples</a></li>
 </ol>
 
+## Terminology
+The terminology used in this specification is based on [xarray's terminology](http://xarray.pydata.org/en/stable/terminology.html), however, no xarray knowledge is assumed in this description. There are also some extensions particular to  the InferenceData case.
+
+* **Variable**: NetCDF-like variables are multidimensional labeled arrays representing a single quantity. Variables and their dimensions must be named. They can also have attributes describing it. Relevant terms related to InferenceData variables are: *variable_name*, *values* (its data), *dimensions*, *coordinates*, and *attributes*.
```

Comment:
"...describing them."

Should we have a sentence describing attributes, as well?  I think they are just an arbitrary dictionary of added information, right?

Reply:
Yes, I will add it whenever I have time